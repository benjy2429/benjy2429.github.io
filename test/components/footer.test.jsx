import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../../src/components/footer';

const socialLinks = [
  {
    name: 'Social link',
    href: 'social.com'
  }
];

describe('Footer component', () => {
  it('renders with social links', () => {
    const component = shallow(<Footer socialLinks={socialLinks} />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
