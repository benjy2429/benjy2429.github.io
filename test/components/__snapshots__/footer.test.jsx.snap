// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders with social links 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <p
      className="footer-title"
    >
      Ben Carr
      <br />
      Software Developer
    </p>
    <div
      className="footer-grid"
    >
      <div
        className="footer-grid-left"
      >
        <h3
          className="footer-links-title"
        >
          Menu
        </h3>
        <ul
          className="footer-links"
        >
          <li>
            <a
              className="link-light"
              href="#work"
            >
              Work
            </a>
          </li>
          <li>
            <a
              className="link-light"
              href="#education"
            >
              Education
            </a>
          </li>
          <li>
            <a
              className="link-light"
              href="#projects"
            >
              Projects
            </a>
          </li>
          <li>
            <a
              className="link-light"
              href="#contact"
            >
              Contact
            </a>
          </li>
        </ul>
      </div>
      <div
        className="footer-grid-left"
      >
        <h3
          className="footer-links-title"
        >
          Social
        </h3>
        <ul
          className="footer-links"
        >
          <li
            key="Social link"
          >
            <a
              className="link-light"
              href="social.com"
            >
              Social link
            </a>
          </li>
        </ul>
      </div>
      <div
        className="footer-grid-right"
      >
        Copyright Â© 
        twenty-eighteen
      </div>
    </div>
  </div>
</footer>
`;
